tests/conftest.py

```diff
@@ -325,6 +325,13 @@ def cluster_teardown_finalizer():
     # Verify health of ceph cluster
     # TODO: move destroy cluster logic to new CLI usage pattern?
     log.info("Done creating rook resources, waiting for HEALTH_OK")
```

Comment:
I would just move this log statement after your patch code so logging the workflow matches what logging is saying.

Reply:
I can add a new logging statement before patch, the HEALTH_OK logging looks accurate to me since we have reached before this point int time?

Reply:
Basically what I want to avoid is if your block of code were to raise an error for any reason, the log would say we were waiting for HEALTH_OK which isn't technically true since we never got to the health check. You can add a log statement in before your patch, that's totally fine. I just wanted to move the one on line 327 after your patch so we don't log that we are waiting for HEALTH_OK before we actually are.

Reply:
Sorry I missed the *waiting* part in the log(was thinking HEALTH_OK has already reached), makes sense to me! , will fix it.